### DISCUSSION TITLE
是否可以隐藏.password文件

### DISCUSSION THREAD

**Discussion by tkfanswind** [spencerwooo/onedrive-vercel-index#120] [General]
现在这个文件.password会显示出来，不太好。
.password可以加密？没有相关说明
感谢

**Comment by York618** [comment]
issue区最适合

**Reply by tkfanswind** [reply]
咋解决？

**Comment by opoet** [comment]
将 `FileListing.tsx` 的第 `231` 行改成
```
<div className={`hover:bg-gray-100 dark:hover:bg-gray-850 grid grid-cols-12 ${c.name == '.password' ? 'hidden': ''}`} key={c.id}>
```
就可以隐藏文件 `.password` 了

**Reply by adhiduzz** [reply]
hi
dont understand, i just using gtranslate, is this code replacing line 231 (removed "setTotalSelected(2)")?
i tried replace but didnt work

**Reply by opoet** [reply]
Unfortunately, the method I described above is applicable to the old version, this method is not valid for the current new version.

**Reply by adhiduzz** [reply]
do you know the code for the latest?

**Reply by issenn** [reply]
> do you know the code for the latest?

try this https://github.com/issenn/onedrive-vercel-index

**Comment by opoet** [comment]
关于加密 #87
